lib/store/index.js

```diff
@@ -7,12 +7,25 @@
 import { createStore, applyMiddleware } from 'redux';
 import reducer from './reducer';
 import thunk from 'redux-thunk';
+import createLogger from 'redux-logger';
```

Comment:
I don't think conditional import of es6 modules is allowed per the spec (confirmed by jonco).